data/public-cms/versions/f2c72f0/version.yml

```diff
@@ -0,0 +1,17 @@
+build:
+  classes: publiccms-parent/publiccms-common/target/classes
+  commands:
+  - cd publiccms-parent
```

Comment:
I think this has no effect, since the commands are executed each in a fresh shell. To `cd` before another command specify one command, such as `cd foo & other-command`.

Reply:
Thanks for this information. I have fixed the corresponding yml-file in commit  1a58711 and verified the results with:
````
mubench> pipeline compile --only public-cms
[INFO   ] Starting benchmark...
[INFO   ] All requirements satisfied. You're good to go.
[INFO   ] Compiling project 'public-cms' version f2c72f0...
````